proxysql/test-proxysql

```diff
@@ -38,45 +38,66 @@ function process_proxysql_output {
+
+# Installing PXC-8.0 client package
+  sudo wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+  sudo dpkg -i percona-release_latest.generic_all.deb
+  sudo percona-release enable-only pxc-80 release
+  sudo percona-release enable tools release
+  sudo apt-get update -y
+  sudo apt-get install -y percona-xtradb-cluster
+
 elif [[ "${PXC_VERSION}" == "PXC57" ]]; then
-    sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz  https://www.percona.com/downloads/Percona-XtraDB-Cluster-57/Percona-XtraDB-Cluster-5.7.31-31.45-3/binary/tarball/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz
-    sudo tar -C ${WORKDIR_ABS}/WORKDIR/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/WORKDIR/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz | head -1)
-    if [ -f /usr/bin/yum ]; then
-        sudo yum remove -y Percona-Server-shared-56-5.6.49-rel89.0.1.el7.x86_64
-        if [[ $RHEL -eq 8 ]]; then
-            sudo yum remove -y mariadb-connector-c-config-3.0.7-1.el8.noarch
-        fi
-        sudo yum install -y percona-xtrabackup-24 Percona-XtraDB-Cluster-57 proxysql2
-    else
-        sudo apt update -y
-        sudo apt install -y percona-xtrabackup-24 percona-xtradb-cluster-client-5.7 proxysql2
-    fi
+# Downloading PXC-5.7 tarball package
+  sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster-5.7.33-rel36-49.1.Linux.x86_64.glibc2.12-minimal.tar.gz  https://www.percona.com/downloads/Percona-XtraDB-Cluster-57/Percona-XtraDB-Cluster-5.7.33-31.49/binary/tarball/Percona-XtraDB-Cluster-5.7.33-rel36-49.1.Linux.x86_64.glibc2.12-minimal.tar.gz
+
+# Installing PXC-5.7 client
+  sudo wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+  sudo dpkg -i percona-release_latest.generic_all.deb
+  sudo percona-release enable original
+  sudo apt-get update -y
+  sudo apt-get install -y percona-xtradb-cluster-57
 fi
 
-tar -C ${WORKDIR_ABS}/WORKDIR/PROXY  --strip-components=1 -zxpf $(ls $WORKDIR_ABS/proxysql*.tar.gz | head -1)
+if [ -f /usr/bin/yum ]; then
+  sudo yum remove -y Percona-Server-shared-56-5.6.49-rel89.0.1.el7.x86_64
+  if [[ $RHEL -eq 8 ]]; then
+    sudo yum remove -y mariadb-connector-c-config-3.0.7-1.el8.noarch
+  fi
+  # TODO: For CentOS machine support both PXC5.7 and 8.0
+    sudo yum install -y percona-xtrabackup-24 Percona-XtraDB-Cluster-57 proxysql2
+else
+# Installing proxysql package
+  sudo apt update -y
+  sudo wget -q -O ${WORKDIR_ABS}/proxysql_2.0.17-ubuntu20_amd64.deb https://repo.proxysql.com/ProxySQL/proxysql-2.0.x/focal/proxysql_2.0.17-ubuntu20_amd64.deb
+  sudo apt-get remove proxysql2
+  sudo apt-get purge proxysql2
```

Comment:
sudo dpkg --purge proxysql2 - should be enough

Reply:
Ok. I will update it